e2e/helpers/acl.ts

```diff
@@ -9,11 +12,15 @@ async function getEntityACL(page: Page, entityId: string, accessToken: string) {
   const acl = await page.evaluate(
     async ({ entityId, accessToken }) => {
       // @ts-expect-error: Cannot find name 'SRC'
-      return await SRC.SynapseClient.getEntityACL(entityId, accessToken)
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
+      return (await SRC.SynapseClient.getEntityACL(
+        entityId,
+        accessToken,
+      )) as AccessControlList
```

Comment:
Defining an object in SWC that represents SRC would be difficult, so ignore these errors for now since we plan to clean this up [once SRC](https://sagebionetworks.jira.com/browse/SWC-6550) can be called directly from the e2e tests.

Reply:
This is still primarily blocked by ESM/CJS mismatch issues, correct?

Reply:
Yes, importing SRC directly is blocked by ESM/CJS issues. However, we'd also considered  moving SynapseClient into its own package that could run in browser or node.js environments, so that we wouldn't have to resolve the ESM/CJS issues for all of SRC. I think this work was blocked by the OpenAPI work.